pytest_twisted.py

```diff
@@ -35,7 +35,7 @@ class AsyncFixtureUnsupportedScopeError(Exception):
     @classmethod
     def from_scope(cls, scope):
         return cls(
-            'Unsupported scope used for async fixture: {}'.format(scope)
+            'Unsupported scope {0!r} used for async fixture'.format(scope)
```

Comment:
So it already was in the first error you shared with me and I still didn't catch it.  auuuhghghh